### PR TITLE
Login Screen Updated

### PR DESCRIPTION
**Pull request by butalas**
### Description
The branch is for the login screen.

### Context
I made the changes to the login screen based on updated designs in Figma and your comments!

### Issue
#23 

### Screenshot
img_6056

### CODE REVIEW

File: screens/login.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
 export default class Login extends React.Component {
   state = {
     fontLoaded: false,
   };
   async componentDidMount() {
     await Font.loadAsync({
-      'Montserrat-Regular': require('/Users/nona/shopu/assets/Montserrat-Regular.otf'),
-      'Montserrat-Medium': require('/Users/nona/shopu/assets/Montserrat-Medium.otf'),
+      'Montserrat-Regular': require('/Users/nona/shopu/assets/fonts/Montserrat-Regular.otf'),
```
Comment (ijemmao) [inline screens/login.js]: Routes to any files or assets in a project should not start with your local computer's home file. The moment I install this app on my phone, the app will look for the font with that path which won't exist.

Instead you should start your path from the current file that you're at - it should look something like `./../assets/fonts/Montserrat-Regular.otf`

**Comment by butalas** [comment]
I just changed the paths of the fonts (I think the PR is done).

**Comment by ijemmao** [comment]
@butalas Do the fonts work in this PR?

**Comment by butalas** [comment]
I just loaded them in `login.js` and it's not working. It only works when I put in the absolute path from my User folder for each font.

**Comment by butalas** [comment]
I actually got it to work! I needed to do `./assets/fonts/Montserrat-Regular.otf`. I'll make another commit with the right changes.

**Comment by butalas** [comment]
Okay so now the font paths are updated correctly and they work. I'm pretty sure this page is done.

**Pull request merged**